warehouse/models/staging/payments/littlepay/stg_littlepay__product_data.sql

```diff
@@ -79,12 +99,24 @@ stg_littlepay__product_data AS (
         scheduled_end_date_time,
         all_day,
         weekly_cap_start_day,
+        weekly_cap_end_day,
         number_of_days_in_cap_window,
         capping_duration,
         number_of_transfer,
         capping_time_zone,
-        capping_overlap,
+        capping_overlap_time,
```

Comment:
This seems to always be null, is this field present in the actual files we're getting?